core/src/Alert.html

```diff
@@ -0,0 +1,99 @@
+<div
+  class="fd-alert fd-alert--{settings.type} {settings.dismissButton ? 'fd-alert--dismissible' : ''}"
+  role="alert"
+  id="j2ALl423"
+  data-cy="luigi-alert"
+>
+  {#if settings.dismissButton}
+  <button
+    class="fd-alert__close close-button"
+    on:click="fire('alertDismiss')"
+    aria-label="Close"
+    aria-controls="j2ALl423"
+    data-cy="luigi-alert-dismiss"
+  ></button>
+  {/if}
+  {@html settings.text}
+</div>
+
+<script type="text/javascript">
+  import * as Routing from './services/routing.js';
+  import * as RoutingHelpers from './utilities/helpers/routing-helpers';
+
+  export default {
+    oncreate() {
+      const defaultSettings = {
+        text:
+          'I am a Luigi alert with extremely relevant information for the user',
+        type: 'info',
```

Comment:
![bildschirmfoto 2019-02-06 um 11 45 27](https://user-images.githubusercontent.com/14272184/52336393-1cece100-2a05-11e9-87f8-0d18e3f8a855.png)

Reply:
Default text not being displayed when not text field is provided is now fixed